tests/minimax.wcon

```diff
@@ -0,0 +1,39 @@
+{
+    "tracker-commons":true,
+    "metadata":{
+           "lab":{"location":"CRB, room 5020", "name":"Behavioural Genomics" },
+           "who":"Firstname Lastname",
+           "timestamp":"2012-04-23T18:25:43.511Z",
+           "temperature":{ "experiment":22, "cultivation":20, "units":"C" },
+           "humidity":{ "value":40, "units":"%" },
+           "dish":{ "type":"petri", "size":35, "units":"mm" },
+           "food":"none",
+           "media":"agarose",
+           "sex":"hermaphrodite",
+           "stage":"adult",
+           "age":"18:25:43.511",
+           "strain":"CB4856",
+           "image_orientation":"imaged onto agar",
+           "protocol":"text description of protocol",
+           "software":{
+                "tracker":{ "name":"Software Name", "version":"1.3.0"},
+                "featureID":"@OMG"
+           },
+           "settings":"Any valid JSON entry with hardware and software configuration can go here"
+     },
+    "units":{"t":"s", "x":"mm", "y":"m"},
+    "data":[
+        {"@OMG":{
```

Comment:
The problem with `[{"@OMG": {}, {"id"=1, "t"=1, "x"=2, "y"=3 }]` is that the data array is no longer an array of comprehensible items.  The problem dropping `["id"=1, "t"=1, "x"=2, "y"=3, "@XJ": {"outline": "0zxqjhFLhqgf20%1fjqFDq75"}]` is that some people (e.g me) need to store extra data on every timepoint.

I suppose we could say that the ID is optional, so you can put things in the data array that aren't animals (and only things with an ID are required to have x and y).  But if you don't even have a time, I'm not sure why it belongs in the data array.  I like flexibility, but I want the format to be constrained enough so you at least have some idea of where to look for which kind of data.